packages/core/scripts/setup/uninstall.ts

```diff
@@ -0,0 +1,305 @@
+import EmbraceLogger from '../../src/logger';
+const fs = require('fs');
+
+import { embraceJSON, getBuildGradlePatchable } from '../util/android';
+import { FileUpdatable } from '../util/file';
+import {
+  bundlePhaseRE,
+  embraceNativePod,
+  embracePlistPatchable,
+  exportSourcemapRNVariable,
+  getPodFile,
+  xcodePatchable,
+} from '../util/ios';
+import Wizard, { Step } from '../util/wizard';
+import { androidEmbraceSwazzlerPlugin, androidGenericVersion } from './android';
+import { findNameWithCaseSensitiveFromPath } from './ios';
+import { SUPPORTED_LANGUAGES } from './patches/common';
+import {
+  getTextToAddWithBreakingLine,
+  SUPPORTED_PATCHES,
+} from './patches/patch';
+
+const packageJson = require('../../../../../../package.json');
+
+const logger = new EmbraceLogger(console);
+
+interface ITextToDelete {
+  ITextToDelete: string | RegExp;
+}
+
+type FindFileFunction = () => FileUpdatable | undefined;
+
+interface IUnlinkEmbraceCode {
+  stepName: string;
+  fileName: string;
+  textsToDelete: ITextToDelete[];
+  findFileFunction: FindFileFunction;
+  docUrl: string;
+}
+
+const UNINSTALL_ANDROID_SWAZZLER_IMPORT: IUnlinkEmbraceCode = {
```

Comment:
not a blocker but I think we should revisit this at some point as there's still opportunity for this to become out of sync with the install steps, I think ideally there would be just be a single list of operations each of which knowing how to do+undo itself